scripts/hid_configurator/modules/dfu.py

```diff
@@ -17,6 +17,13 @@
 from NrfHidDevice import EVENT_DATA_LEN_MAX
 import imgtool.image
 
+try:
+    from suit_generator.envelope import SuitEnvelope
```

Comment:
Could we squash commits related to HID configurator? The second commit removes most of the changes applied by the first one